lnbits/core/views/public_api.py

```diff
@@ -20,7 +20,7 @@ async def api_public_payment_longpolling(payment_hash):
         raise HTTPException(
             status_code=HTTPStatus.NOT_FOUND, detail="Payment does not exist."
         )
-    elif not payment.pending:
+    if payment.success:
         return {"status": "paid"}
```

Comment:
"paid" is not a valid value for status with the changes in this PR.
```suggestion
      return {"status": payment.status}
```

The LNbits funding source should also be correlated.